x/distribution/spec/05_hooks.md

```diff
@@ -5,26 +5,44 @@ order: 5
 # Hooks
 
 ## Create or modify delegation distribution
- 
+
  - triggered-by: `staking.MsgDelegate`, `staking.MsgBeginRedelegate`, `staking.MsgUndelegate`
 
-The pool of a new delegator bond will be 0 for the height at which the bond was
-added, or the withdrawal has taken place. This is achieved by setting
-`DelegationDistInfo.WithdrawalHeight` to the height of the triggering transaction. 
+### Before
+
+The delegation rewards are withdrawn to the delegator's withdraw address.
```

Comment:
```suggestion
The delegation rewards are withdrawn to the delegators' withdraw address.
```

Reply:
In this case, it's singular.

Reply:
In my suggestion it's also singular. It's possessive form.

Reply:
I'm not a native speaker, but I thought `delegator's` was possessive singular, `delegators'` was possessive plural. I might be wrong.
But if there's any chance of misunderstanding we can do better. How about this:
```
The delegation rewards are withdrawn to the withdraw address of the delegator.
```

Reply:
I like it! @hjorthjort writing around questionable wording and punctuation is known in the technical writing world as the "mongoose principle" one day we can chat about this fun conundrum!

Reply:
I like the suggestion as well. Let's use it.